model-optimizer/unit_tests/extensions/front/tf/TFSliceToSlice_test.py

```diff
@@ -34,7 +34,6 @@ def test_slice_replacer_begin_with_2_inputs(self):
         graph = build_graph(nodes_attrs=nodes, edges=[
             ('input', 'tfslice'),
             *connect_front('begin:0', '1:tfslice'),
-            *connect_front('begin:0', '0:john_doe'),
```

Comment:
@pavel-esir , please, take a look at the changes in this file.

Reply:
Why do we need the node `john_doe` without output edges?

Reply:
This test was written a long time ago, I was incorrectly reconnecting begin/end inputs without general orthodox port api. To debug that and prevent form future failing I added this test. But now in TFSliceToSlice general mechanism so with can remove the whole `test_slice_replacer_begin_with_2_inputs` case without any harm.

Reply:
The test case `test_slice_replacer_begin_with_2_inputs` was deleted.